### PR TITLE
code of conduct added

### LINKED ISSUE #76: Add CODE_OF_CONDUCT.md file
Add a new file namely CODE_OF_CONDUCT.md that contains the standards, scope and other responsibilities like reporting violations and conflict resolution.

For reference see: https://github.com/tensorflow/tensorflow/blob/master/CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by shreyakapoor08**
Fixes #76 

Code of Conduct content added

Files added:
- CODE_OF_CONDUCT.md

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* Disrespecting an individual on the basis of gender, religion or culture.
+* Publishing private details of people, such as physical or email addresses, without their prior consent
+* Undertaking the action of insider trading
+
+Our Responsibilities
+Authorities are responsible for creating an environment where everyone is treated equally and strict actions should be taken in case of unacceptable behaviour.
+Authorities have the duty and responsibility to delete, modify, or reject comments, code, wiki updates, problems, and other contributions that are not consistent with this Code of Conduct, or temporarily or permanently ban any user for other actions that they find unacceptable, disruptive, offensive, or harmful.
+
+Scope
+This code of conduct is applicable to all the content on Book Uploader website, Book Uploader githubs organisation and all other websites which are linked to it whether offline or online.
```
Comment (rkpattnaik780) [inline CODE_OF_CONDUCT.md]: We don't have a github organization for BUB2.

**Comment by shreyakapoor08** [comment]
Okay I will just correct that, any more corrections required apart from this?

**Comment by rkpattnaik780** [comment]
> Okay I will just correct that, any more corrections required apart from this?

Only this much I guess.

**Comment by shreyakapoor08** [comment]
@rkpattnaik780  changes done kindly check

**Comment by rkpattnaik780** [comment]
@shreyakapoor08 there are conflicts that need to be resolved.

**Comment by shreyakapoor08** [comment]
> @shreyakapoor08 there are conflicts that need to be resolved. Seems you had rebased with `master` and not `develop`

yes i guess i did it with master, can you please tell me what should I do now ?

**Comment by rkpattnaik780** [comment]
> > @shreyakapoor08 there are conflicts that need to be resolved. Seems you had rebased with `master` and not `develop`
> 
> yes i guess i did it with master, can you please tell me what should I do now ?

Feel free to DM me on slack. Slack name - Ramakrishna Pattnaik

**Comment by rajat2502** [comment]
Cool.

**Pull request merged**